boards/common/sodaq/Makefile

```diff
@@ -1,3 +1,8 @@
 MODULE = boards_common_sodaq
 
+ifneq (,$(filter boards_common_samd21-arduino-bootloader,$(USEMODULE)))
```

Comment:
This block must also be added in boards/sodaq-one and boards/sodaq-sara-aff to fix the build failures on Murdock.